api/src/main/java/org/openmrs/BaseOpenmrsMetadata.java

```diff
@@ -22,29 +25,40 @@
  * @since 1.5
  * @see OpenmrsMetadata
  */
+@MappedSuperclass
 public abstract class BaseOpenmrsMetadata extends BaseOpenmrsObject implements OpenmrsMetadata {
 	
 	//***** Properties *****
+	@Column(name = "name", nullable = false, length = 255)
 	@Field
 	private String name;
 	
+	@Column(name = "description", length = 255)
```

Comment:
Then i guess we can leave it as 255 here and override it in the concrete classes